static/js/dev.ts

```diff
@@ -18,33 +18,36 @@
  * @fileoverview dev page.
  */
 
+import React from "react";
+import ReactDOM from "react-dom";
 import { DataPoint, DataGroup } from "./chart/base";
+import { ChartRegionPropsType, ChartRegion } from "./timeline_chart";
 
 import {
   drawSingleBarChart,
   drawStackBarChart,
   drawGroupBarChart,
   drawLineChart,
   drawGroupLineChart,
+  computePlotParams,
 } from "./chart/draw";
 
-window.onload = () => {
+window.onload = function () {
```

Comment:
can you leave this and the other parts of this file unchanged, they are updated as a lint fix in previous PR

Reply:
Done